mis_builder/models/kpimatrix.py

```diff
@@ -462,7 +461,13 @@ def as_dict(self):
             for cell in row.iter_cells():
                 if cell is None:
                     # TODO use subcol style here
-                    row_data['cells'].append({})
+                    col_data = {
+                        'val': None,
+                        'val_r': '',
+                        'val_c': '',
+                        'style': None,
+                    }
+                    row_data['cells'].append(col_data)
```

Comment:
Is this change necessary? If yes why?

Reply:
Yes. This fix prevented errors raising when you attempted to create lines.